### DISCUSSION TITLE
WF Task: Create address book

### DISCUSSION THREAD

**Discussion by IanMuller** [YetiForceCompany/YetiForceCRM#14538] [How to]
Create address book - What actually happening when this task is executed? Cannot find a word about it ..

**Comment by mariuszkrzaczkowski** [comment]
Address book for roundcube
![image](https://user-images.githubusercontent.com/10198654/114343642-9d1bad80-9b5e-11eb-8622-34aba15cca09.png)

**Reply by Giorgos-Hellene** [reply]
these are contacts' emails
